test/NameBinding/import-transitive-cross-imports.swift

```diff
@@ -0,0 +1,23 @@
+// This file tests that re-exports can load a cross-import overlay.
+
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/CrossImport/ %t/
+// RUN: %{python} %S/Inputs/rewrite-module-triples.py %t %module-target-triple
+
+// RUN: %target-typecheck-verify-swift -enable-cross-import-overlays -I %t/include -I %t/lib/swift -F %t/Frameworks -DDIRECT_FIRST
+// RUN: %target-typecheck-verify-swift -enable-cross-import-overlays -I %t/include -I %t/lib/swift -F %t/Frameworks -DDIRECT_SECOND
+
+#if DIRECT_FIRST
+import ThinLibrary
```

Comment:
I should use DeclaringModule and BystandingModule in this test instead—they’re more “real”.

Reply:
No, it works better this way—the AlwaysImportedLibrary setup has a way to run negative tests.